### ISSUE TITLE
hCAPTCHA prompt a11y evaluation (round 2)

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#7060]
The main tasks are:
- [x] Coordinate a11y fixes with hCAPTCHA 
- [x] Request an a11y evaluation
- [ ] Review the a11y evaluation and decide what to do next

This ticket can be used to track progress/issues.

**Comment by anikbrazeau** [comment]
https://github.com/cds-snc/platform-forms-client/pull/5411

**Comment by Abi-Nada** [comment]
Sent over documentation to AAACT. Pending response from them (previously took around ~3 months).

**Comment by thiessenp-cds** [comment]
AAACT sent an email that the work is now scheduled. I suspect it will be done (reviewed) in a few months.

**Comment by thiessenp-cds** [comment]
AAACT completed the second evaluation. hCaptcha did much better but sadly still failed with 6 issues.
Attached is the summary and a zip containing more detail.

<img width="1437" height="1051" alt="Image" src="https://github.com/user-attachments/assets/766cedc4-ec89-4944-9ae8-9ba0c8a7d9ed" />

[wcag audit 2.zip](https://github.com/user-attachments/files/30746286/wcag.audit.2.zip)

**Comment by anikbrazeau** [comment]
Aim: one day hopefully we'll be able to use the accessible challenge

**Comment by thiessenp-cds** [comment]
A new ticket was created to follow the next round of testing #7811
